tools/buildmgr/cbuildgen/installer/create_installer.sh

```diff
@@ -73,7 +73,7 @@ cp ../../docs/LICENSE.txt ${distdir}
 # Get cpackget
 cpackget_version="0.2.0"
 cpackget_base=https://github.com/Open-CMSIS-Pack/cpackget/releases/download/v${cpackget_version}/cpackget_${cpackget_version}
-curl --retry 3 -L ${cpackget_base}_windows_amd64.zip   -o temp.zip && unzip -p temp.zip '*cpackget.exe' > ${distdir}/bin/cpackget.exe && rm temp.zip
+curl --retry 3 -L ${cpackget_base}_windows_amd64.zip   -o temp.zip && unzip -p temp.zip '*/cpackget.exe' > ${distdir}/bin/cpackget.exe && rm temp.zip
```

Comment:
Didn't `*` match `/` ? I may have overlooked this case

Reply:
In my local windows environment with unzip 6.00 I got the following error:
```
$ unzip -p temp.zip '*cpackget.exe'
caution: filename not matched:  *cpackget.exe
```